app/stacks/uk-south/appeals-service/providers.tf

```diff
@@ -2,11 +2,14 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "3.107.0"
+      version = "=4.6.0"
+    }
+    time = {
+      source  = "hashicorp/time"
+      version = "~>0.9"

```

Comment:
I'm confused how it worked before!

Reply:
My thinking was, as this is the actual 'project' it should be pinned, and then anything it is calling should be left more open. But I get confused as we are calling components from here, and then we're calling modules from components...